app/assets/stylesheets/_conditionals.scss

```diff
@@ -0,0 +1,40 @@
+// Media query helpers. These make producing IE layouts
+// super easy.
+
+// These are desktop and down media queries
+
+// There is also a local version of this in Smartanswers.
+
+$is-ie: false !default;
+
+@mixin media-down($size: false, $max-width: false, $min-width: false) {
+  @if $is-ie == false {
+    @if $size == mobile {
+      @media (max-width: 640px) {
+        @content;
+      }
+    } @else if $size == tablet {
+      @media (max-width: 800px) {
+        @content;
+      }
+    }
+  }
+}
+
+// mixins from toolkit, only used now by static and whitehall
+
+@mixin ie-lte($version) {
```

Comment:
Now looking a bit deeper... I can't see where this mixin is used, is it needed?

Reply:
Removed